packages/@tinacms/cli/src/cmds/compile/index.ts

```diff
@@ -41,74 +44,107 @@ export const resetGeneratedFolder = async () => {
 
 export const compile = async (_ctx, _next) => {
   logger.info(logText('Compiling...'))
-  // FIXME: This assume it is a schema.ts file
-  if (
-    !fs.existsSync(tinaPath) ||
-    !fs.existsSync(path.join(tinaPath, 'schema.ts'))
-  ) {
+  let schemaExists = true
+  try {
+    getSchemaPath({ projectDir: tinaPath })
+  } catch (_) {
```

Comment:
```suggestion
  } catch {
```
this syntax works if you don't need to do anything with the thrown error